dcpy/test/connectors/test_github.py

```diff
@@ -3,7 +3,6 @@
 TEST_REPO = "data-engineering"
 
 
-## TODO - these tests don't belong here since this functionality has been moved to dcpy
```

Comment:
completed via this file being in `dcpy/test/`, it used to be in the QA app codebase